requirements.txt

```diff
@@ -15,7 +15,13 @@ PyMySQL==0.6.2
 DBUtils==1.1
 pyproj==1.9.3
 prettytable==0.7.2
+
 # When updating nupic.bindings, also update any shared dependencies to keep
 # versions in sync.
 nupic.bindings==0.5.1
 numpy==1.11.2
+
+# for documentation
+sphinx==1.5.3
+sphinx-autobuild==0.6.0
```

Comment:
@scottpurdy @oxtopus I'm thinking about removing the sphinx requirements. No need for most users to have this installed.